pkg/scheduler/apis/config/v1alpha1/BUILD

```diff
@@ -17,7 +17,6 @@ go_library(
         "//pkg/kubelet/apis:go_default_library",
         "//pkg/master/ports:go_default_library",
         "//pkg/scheduler/apis/config:go_default_library",
-        "//staging/src/k8s.io/apimachinery/pkg/apis/config/v1alpha1:go_default_library",
```

Comment:
i don't think this PR should be touching pkg/scheduler.
is this the result of ./hack/update-bazel.sh?

Reply:
yeah. I think so. I have just updated this. I don't know how it got touched. I caught this too. :thinking:

Reply:
Sadly I squashed because I did another rebase on master and was not thinking about leaving the new commits there. my bad.